cardinal/server/handler/query.go

```diff
@@ -18,35 +20,15 @@ import (
 //	@Success      200         {object}  object  "Results of the executed query"
 //	@Failure      400         {string}  string  "Invalid request parameters"
 //	@Router       /query/{queryGroup}/{queryName} [post]
-func PostQuery(queries map[string]map[string]engine.Query, wCtx engine.Context) func(*fiber.Ctx) error {
+func PostQuery(world servertypes.ProviderWorld) func(*fiber.Ctx) error {
 	return func(ctx *fiber.Ctx) error {
-		query, ok := queries[ctx.Params("group")][ctx.Params("name")]
-		if !ok {
-			return fiber.NewError(fiber.StatusNotFound, "query name not found")
-		}
-
 		ctx.Set("Content-Type", "application/json")
-		resBz, err := query.HandleQueryRaw(wCtx, ctx.Body())
-		if err != nil {
+		resBz, err := world.HandleQuery(ctx.Params("name"), ctx.Body())
```

Comment:
Ah I see, ok I'll flip it back.

Reply:
I think it's the other way around. WithCustomQueryGroup allows identical groups but not identical names. Registered queries are stored in a map by name and the group exists as a property on the query itself. 

See: https://github.com/Argus-Labs/world-engine/blob/8ad754dec16e084fad40db77b253d45cea8dc3d8/cardinal/query_manager.go#L27-L37

and: https://github.com/Argus-Labs/world-engine/blob/8ad754dec16e084fad40db77b253d45cea8dc3d8/cardinal/query.go#L35-L43

I could be mistaken here. Let me know how you want to proceed.

Reply:
Thats just an invalid behavior with isQueryNameUnique. As I previously mentioned, just make it such that there is making of group -> name -> query. That's the intended behavior [](edited-with-Axolo)

Reply:
I'm more referring to this:

```
m.registeredQueries[name] = query 
```

the storage for queries makes it so all queries are indexed by a unique name. If we want to change this I recommend  moving this to another ticket as it will change how things work under the hood. This context thing is more of an aesthetic api change and doing this imo will make the changes in this PR too big and less focused. 

I think this could be a good plan: I'll change the url and method signatures back to include group, and I'll create a ticket for changing the whole thing to `group->name->query` for the long term. 

Let me know if you're cool with the above.

Reply:
scott said this is ok in an in person discussion.

Reply:
This is done.